db/migrate/20240607134321_add_one_login_uid_to_candidates.rb

```diff
@@ -0,0 +1,5 @@
+class AddOneLoginUidToCandidates < ActiveRecord::Migration[7.1]
+  def change
+    add_column :candidates, :one_login_uid, :string
```

Comment:
I'd err on the side of explicitness here.

```suggestion
    add_column :candidates, :govuk_one_login_uid, :string
```